### ISSUE TITLE
Inference time is longer than training time

### ISSUE THREAD

**Issue by Qusijia** [lucidrains/DALLE2-pytorch#301]
My training time is much shorter than my reasoning time. Would you like to inquire if this situation is normal? Or what advice can you give me.

Training Code：
        loss = priorTrainer.diffusion_prior(text_embed=text_embed, image_embed=image_embed, text_encodings=None)
        priorTrainer.optimizer.zero_grad()
        loss.backward()
        priorTrainer.optimizer.step()
        priorTrainer.scheduler.step()

Inference code：
      prior.sample_add( text_embed = text_embed)

**Comment by AIDevMonster** [comment]
sorry, I'm busy now.
2 days late, I will send you.
thx.
Paul
